attestation.go

```diff
@@ -47,8 +47,7 @@ bW5yWvyS9zNXaqGaUmP3U9/b6DlHdDogMLu3VLpBB9bm5bjaKWWJYgWltCVgUbFq
 Fqyi4+JE014cSgR57Jcu3dZiehB6UtAPgad9L5cNvua/IWRmm+ANy3O2LH++Pyl8
 SREzU8onbBsjMg9QDiSf5oJLKvd/Ren+zGY7
 -----END CERTIFICATE-----
-`)
-)
+`)}
```

Comment:
This will only work with Yubikey's issued after 2018. Can you add the old CA cert as well? https://developers.yubico.com/PIV/Introduction/piv-attestation-ca-old.pem